packages/manager/apps/hpc-vmware-managed-vcd/src/components/tiles/organization-general-information-tile/OrganizationGeneralInformationTile.component.tsx

```diff
@@ -39,72 +33,86 @@ export default function OrganizationGenerationInformationTile({
   const navigate = useNavigate();
 
   return (
-    <OsdsTile className="w-full h-full flex-col" inline rounded>
-      <div className="flex flex-col w-full">
-        <CommonTitle>
-          {t('managed_vcd_dashboard_general_information')}
-        </CommonTitle>
-        <OsdsDivider separator />
-        <TileSubtitle>{t('managed_vcd_dashboard_name')}</TileSubtitle>
-        <div className="flex flex-row justify-between items-center">
-          <Description>{vcdOrganization?.currentState?.fullName}</Description>
-          <OsdsIcon
-            aria-label="edit"
-            className="mx-6 cursor-pointer"
-            name={ODS_ICON_NAME.PEN}
-            size={ODS_ICON_SIZE.xxs}
-            color={ODS_THEME_COLOR_INTENT.primary}
-            onClick={() => navigate(urlParts.editName)}
-          />
-        </div>
-        <OsdsDivider separator />
-        <div className="flex flex-col mb-3">
-          <TileSubtitle>{t('managed_vcd_dashboard_description')}</TileSubtitle>
-          <div className="flex flex-row justify-between items-center">
-            <Description>
-              {vcdOrganization?.currentState?.description}
-            </Description>
-            <OsdsIcon
-              aria-label="edit"
-              className="mx-6 cursor-pointer"
-              name={ODS_ICON_NAME.PEN}
-              size={ODS_ICON_SIZE.xxs}
-              color={ODS_THEME_COLOR_INTENT.primary}
-              onClick={() => navigate(urlParts.editDescription)}
+    <DashboardTile
+      title={t('managed_vcd_dashboard_general_information')}
+      items={[
+        {
+          id: 'name',
+          label: t('managed_vcd_dashboard_name'),
+          value: (
+            <div className="flex justify-between items-center">
+              <Description>
+                {vcdOrganization?.currentState?.fullName}
+              </Description>
+              <OsdsIcon
+                aria-label="edit"
+                className="mx-6 cursor-pointer"
+                name={ODS_ICON_NAME.PEN}
+                size={ODS_ICON_SIZE.xxs}
+                color={ODS_THEME_COLOR_INTENT.primary}
+                onClick={() => navigate(urlParts.editName)}
+              />
```

Comment:
**niptick** : could be defined as a dedicated component as it's used, a least, twice.